DevTools/TermTranslator/tmxFormat.cs

```diff
@@ -0,0 +1,144 @@
+﻿using System;
```

Comment:
Go ahead and rename this file to TmxFormat for consistency and conformance to standards.